ui/src/testing/stubs/config-api-stub.ts

```diff
@@ -1,15 +1,11 @@
 import { ConfigApi, ConfigResponse } from 'generated/fetch';
 
 import defaultServerConfig from 'testing/default-server-config';
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class ConfigApiStub extends ConfigApi {
   public config: ConfigResponse;
 
   constructor(configOverrides: Partial<ConfigResponse> = {}) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.